src/Slider.jsx

```diff
@@ -308,15 +308,15 @@ class Rheostat extends React.Component {
       : this.handleNode.clientWidth;
   }
 
-  getSnapPosition(positionPercent) {
+  getSnapPosition(positionPercent, ignoreSnap = false) {
```

Comment:
Can we leave a comment for why we're adding this argument?

Also just in case, @josh-perez, do you happen to remember how this worked before? It kind of seemed like `getSnapPosition` was being using to determine where the progress bar would slide to... except it would always give the snap-point not the drag position.